crates/rattler_conda_types/src/repo_data/mod.rs

```diff
@@ -51,16 +51,16 @@ pub struct RepoData {
         rename = "packages.conda",
         serialize_with = "sort_map_alphabetically"
     )]
-    pub conda_packages: FxHashMap<String, PackageRecord>,
+    pub conda_packages: HashMap<String, PackageRecord, RandomState>,
```

Comment:
If the order matters, which it does in this case, I think we should use a different map. We can also use an `OrderMap<String, PackageRecord, ahash::RandomState>`.

Reply:
We can also remove the `serialize_with` part if the insertion order is kept.

Reply:
So it should be fine in this case, as `RepoData` isn't hashable.